src/privileged/trackers/api.js

```diff
@@ -23,11 +25,11 @@ function getMostRecentBrowserWindow() {
 }
 
 class TrackersEventEmitter extends EventEmitter {
-  emitTrackersExist() {
-    this.emit("trackers-exist");
+  emitTrackersExist(tabId) {
+    this.emit("trackers-exist", {tabId});
   }
-  emitlocationChange() {
-    this.emit("location-change");
+  emitlocationChange(tabId) {
```

Comment:
nit typo on my part - could you change all the `emitlocationChange` to `emitLocationChange`